### DISCUSSION TITLE
How to disble Advance deployment protection and preview deployment suffix.

### DISCUSSION THREAD

**Discussion by mohammedsalmanneosme** [vercel/community#3300] [Feature Question]
### Summary

![image (20)](https://github.com/vercel/community/assets/77548963/8900ddbe-d25a-4880-8ff0-b2e40b50c04b)

By mistake i added Advance deployment protection and preview deployment suffix while upgrade it asked billing that. Now i do not need those. 

### Steps to Reproduce

1. Trying upgrade pro plan

**Comment by amyegan** [comment]
Hey @mohammedsalmanneosme. To [opt out of Advanced Deployment Protection](https://vercel.com/docs/concepts/deployments/advanced-deployment-protection#disable-advanced-deployment-protection), go to your Team Settings → Billing page. You can disable Advanced Deployment Protection in the Add-Ons section of that page.

**Reply by mohammedsalmanneosme** [reply]
@amyegan But I cannot see the option in my Team settings → Billing page. What i do to see the options?

**Reply by amyegan** [reply]
When logged in to vercel.com, use the dropdown at the top of the page to select the appropriate team. Then, click "Settings" to access the team settings.

<img width="744" alt="Team dashboard navigation bar with settings tab active" src="https://github.com/vercel/community/assets/8247105/41b8eb73-7771-476e-a6a5-40464bcab397">

From there, click "Billing" in the side menu to show the team's billing settings. Scroll down until you see the "Add-Ons" section which includes a toggle for "Advanced Deployment Protection". Use the toggle to disable this feature for the team.

<img width="776" alt="Add on section on the billing settings page" src="https://github.com/vercel/community/assets/8247105/b881841f-0671-4b30-bd8a-524570039f2f">

**Reply by mohammedsalmanneosme** [reply]
@amyegan Still I cannot see these option my dashboard. Currently I am Pro trial . Is this the reason I am not getting those options. For your information My role is owner in my dashboard. Thank you

![image](https://github.com/vercel/community/assets/77548963/74f938ce-a01f-4e2e-aea3-18debd6af025)

**Reply by mohammedsalmanneosme** [reply]
@amyegan DO you have any idea about above one?

**Reply by amyegan** [reply]
Sorry for the delay! I don't often handle team plan upgrades or billing questions, so I need to ask some of my colleagues about this. 🙇 

I generally recommend [submitting a case](https://vercel.com/help#issues) to the Customer Success Team for help with Pro team usage and billing questions since that will get you in contact with the folks who specialize in those topics. But I asked them about this situation and I'll let you know when I learn more.

**Reply by amyegan** [reply]
I confirmed that the team will be able to help you remove the feature, but they asked that you [open a case](https://vercel.com/help#issues) for this request 🙏

**Reply by mohammedsalmanneosme** [reply]
Thank you @amyegan

**Reply by mohammedsalmanneosme** [reply]
@amyegan I submit a case 1 day ago. still no response. I trial goes to end today. I need to upgrade pro without Add-ons
![image](https://github.com/vercel/community/assets/77548963/af72c186-ff82-4653-89f1-49232e333a12)
